device/mellanox/x86_64-mlnx_msn4600-r0/ACS-MSN4600/sai_4600.xml

```diff
@@ -0,0 +1,471 @@
+<?xml version="1.0"?>
```

Comment:
please send the file to Itai to confirm it. Not sure if he has one to use on SAI github but at least  he need to confirm it

Reply:
We talked with Itay which told us that the file is already in github which I found here:
https://github.com/Mellanox/SAI-Implementation/blob/80937117fcd54e8df0581368f652f3664af6aecb/mlnx_sai/src/sai_4600.xml

But he did say it was a draft file.